src/ctia/bundle/routes.clj

```diff
@@ -14,6 +14,7 @@
 
 (s/defschema BundleExportQuery
   {:ids [s/Str]
+   (s/optional-key :related_to) [(s/enum :source_ref :target_ref)]
```

Comment:
A simple remark, the actual schema authorize duplicate related_to values. If that isn't a problem for swagger/compojure-api, perhaps we could use a set instead of a list here.